userguide/payment/includes/payment-methods-flows.adoc

```diff
@@ -2,17 +2,18 @@ A *payment method* refers to the end-user's payment details. When an end-user en
 
 The https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L541[PaymentApi#addPaymentMethod] is the entry point for saving a payment method in Kill Bill. Thus, a merchant website needs to invoke this method in order to save the payment details as a payment method in Kill Bill.
 
+[[payment_method_flows]]
 === Payment Method Flows Overview
 
 *Payment method flows* describe how a payment method gets created in Kill Bill.
 
 Kill Bill supports three main payment method flows as follows:
 
-* *Client-side tokenization* - In this case, the end user's payment method details are converted into a token which is sent to Kill Bill.
+* *Client-side Tokenization* - In this case, the end user's payment method details are converted into a token which is sent to Kill Bill.
 
-*  *Hosted Payment Page* - In this case, the merchant website redirects the end-user to a page on the payment gateway which collects the payment method details. Kill Bill then invokes the payment gateway to obtain a gateway specific token.
+*  *Hosted Payment Page* - In this case, the merchant website redirects the end-user to a page on the payment gateway which collects the payment method details. Kill Bill then fetches a gateway specific token from the payment gateway.
```

Comment:
```suggestion
*  *Hosted Payment Page* – In this case, the merchant website redirects the end-user to a page on the payment gateway, which collects the payment method details. Kill Bill then fetches a gateway-specific token from the payment gateway.
```
Another way to phrase the first sentence without that comma I added: In this case, the merchant website redirects the end-user to a page on the payment gateway that collects the payment method details. But I think the original, with "which" and a comma, is more clear.

Reply:
Agreed, I've used the original one with "which" and a comma.